dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -0,0 +1,263 @@
+    private static final Set<String> MP22_FEATURE_SET = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MP22_FEATURES_ARRAY)));
+    private static final Set<String> MP30_FEATURE_SET = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MP30_FEATURES_ARRAY)));
+    private static final Set<String> MP32_FEATURE_SET = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MP32_FEATURES_ARRAY)));
+    private static final Set<String> MP33_FEATURE_SET = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MP33_FEATURES_ARRAY)));
+    private static final Set<String> MP40_FEATURE_SET = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(MP40_FEATURES_ARRAY)));
+
+    public static final FeatureSet MP10 = new FeatureSet(MP10_ID, MP10_FEATURE_SET);
+    public static final FeatureSet MP12 = new FeatureSet(MP12_ID, MP12_FEATURE_SET);
+    public static final FeatureSet MP13 = new FeatureSet(MP13_ID, MP13_FEATURE_SET);
+    public static final FeatureSet MP14 = new FeatureSet(MP14_ID, MP14_FEATURE_SET);
+    public static final FeatureSet MP20 = new FeatureSet(MP20_ID, MP20_FEATURE_SET);
+    public static final FeatureSet MP21 = new FeatureSet(MP21_ID, MP21_FEATURE_SET);
+    public static final FeatureSet MP22 = new FeatureSet(MP22_ID, MP22_FEATURE_SET);
+    public static final FeatureSet MP30 = new FeatureSet(MP30_ID, MP30_FEATURE_SET);
+    public static final FeatureSet MP32 = new FeatureSet(MP32_ID, MP32_FEATURE_SET);
+    public static final FeatureSet MP33 = new FeatureSet(MP33_ID, MP33_FEATURE_SET);
+    public static final FeatureSet MP40 = new FeatureSet(MP40_ID, MP40_FEATURE_SET);
+
+    public static final FeatureSet LATEST = MP40;
+
+    public static final FeatureSet[] ALL_SETS_ARRAY = { MP10, MP12, MP13, MP14, MP20, MP21, MP22, MP30, MP32, MP33, MP40 };
+    public static final Set<FeatureSet> ALL = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(ALL_SETS_ARRAY)));
+
+    /**
+     * Get a RepeatTests instance for the given FeatureSets. All FeatureSets will be run in LITE mode.
+     *
+     * @param  server           The server to repeat on
+     * @param  firstFeatureSet  The first FeatureSet
+     * @param  otherFeatureSets The other FeatureSets
+     * @return                  a RepeatTests instance
+     */
+    public static RepeatTests repeat(String server, FeatureSet firstFeatureSet, FeatureSet... otherFeatureSets) {
+        return repeat(server, TestMode.LITE, ALL, firstFeatureSet, otherFeatureSets);
+    }
+
+    /**
+     * @param  server
+     * @param  otherFeatureSetsTestMode
+     * @param  allFeatureSets
+     * @param  firstFeatureSet
+     * @param  otherFeatureSets
+     * @return
+     */
+    public static RepeatTests repeat(String server, TestMode otherFeatureSetsTestMode, Set<FeatureSet> allFeatureSets, FeatureSet firstFeatureSet, FeatureSet... otherFeatureSets) {
```

Comment:
Javadoc (particularly important here as the meaning of `allFeatureSets` and `otherFeatureSetsTestMode` isn't particularly intuitive)

Reply:
done